Scripts/Editor/NodeEditorWindow.cs

```diff
@@ -77,7 +77,15 @@ public partial class NodeEditorWindow : EditorWindow {
         void OnFocus() {
             current = this;
             ValidateGraphEditor();
-            if (graphEditor != null && NodeEditorPreferences.GetSettings().autoSave) AssetDatabase.SaveAssets();
+            if (graphEditor != null)
+            {

```

Comment:
Please use consistent brace formatting (K&R style)

Reply:
Ah yes sorry about that! Should be fixed now